prisma/schema.prisma

```diff
@@ -1,9 +1,6 @@
 generator client {
-  provider = "prisma-client-js"
-}
-
-generator nexusPrisma {
-  provider = "nexus-prisma"
```

Comment:
Could you add back the `nexusPrisma` generator ?

Reply:
`nexus-prisma` doesn't exist on any repo than gql so it causes errors while generating client, etc.

Reply:
ok this is a problem... If we remove that then the gql repo won't work anymore
I have to figure this out